flatpak/manifest_tool/tests/test_flutter_ops.py

```
@@ -541,3 +623,330 @@ def test_bundle_app_archive_no_lotti_module(make_document, tmp_path):
+    assert result.changed
+    sources = lotti["sources"]
+    assert len(sources) == 3  # Original 2 + mimalloc
+
+    # Original sources still there
+    assert sources[0]["type"] == "git"
+    assert sources[1]["type"] == "file"
+    # New mimalloc source
+    assert sources[2]["dest-filename"] == "mimalloc-2.1.2.tar.gz"
+
+
+def test_remove_network_from_build_args(make_document):
```

Ruff (0.13.1)

Redefinition of unused `test_remove_network_from_build_args` from line 68

(F811)